domain/users/user_dto.go

```diff
@@ -0,0 +1,67 @@
+package users
+
+import (
+	"moku-moku/utils/errors"
+	"regexp"
+	"strings"
+)
+
+type User struct {
+	Email      string `json:"email"`
+	Username   string `json:"username"`
+	Name       string `json:"name"`
+	Surname    string `json:"surname"`
+	Biography  string `json:"biography"`
+	Birthday   string `json:"birthday"`
```

Comment:
Will we then have to format it if we leave it as a string? Is there no other way to implement the timestamp?

Reply:
I think we can use either timestamp or html5 format string.

Reply:
The back-end will receive this in a string format from front-end client POST request, but for the database this field will be a timestamp ✨